android/src/main/java/com/swmansion/rnscreens/ScreenTransitionProgressEvent.java

```diff
@@ -0,0 +1,41 @@
+package com.swmansion.rnscreens;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.uimanager.events.Event;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class ScreenTransitionProgressEvent extends Event<ScreenAppearEvent> {
+
+  public static final String EVENT_NAME = "topTransitionProgress";
```

Comment:
Is there a reason to call it `topTransitionProgress` instead of `onTransitionProgress`?